lib/core/seleniumRunner.js

```diff
@@ -7,7 +7,8 @@ const log = require('intel').getLogger('browsertime'),
   isAndroidConfigured = require('../android').isAndroidConfigured,
   UrlLoadError = require('../support/errors').UrlLoadError,
   BrowserError = require('../support/errors').BrowserError,
-  getViewPort = require('../support/getViewPort');
+  getViewPort = require('../support/getViewPort'),
+  ScriptType = require('../support/browserScript').ScriptType;
```

Comment:
If you make this one first (in the bunch starting with `../support`), you change fewer lines and keep alphabetical ordering (by file).